demos/common/python/openvino/model_zoo/model_api/models/model.py

```diff
@@ -84,10 +107,45 @@ def available_wrappers(cls):
 
     @classmethod
     def parameters(cls):
+        '''Defines the description and type of configurable data parameters for the wrapper.
+
+        See `types.py` to observe available types of the data parameter. For each parameter
```

Comment:
```suggestion
        See `types.py` to know about available types of the data parameter. For each parameter
```
Maybe I'm wrong, but `observe` has slightly different meaning, and in this context `know` is more apropriate.

Reply:
```suggestion
        See `types.py` to find available types of the data parameter. For each parameter
```